File: backend/server/router/member/member/sms_log.go
package member

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type SmsLogRouter struct{}

// InitSmsLogRouter 初始化短信日志管理路由信息
func (s *SmsLogRouter) InitSmsLogRouter(Router *gin.RouterGroup, PublicRouter *gin.RouterGroup) {
	smsLogGroup := Router.Group("member").Group("smsLog")
	smsLogRouter := smsLogGroup.Use(middleware.OperationRecord())
	smsLogRouterWithoutRecord := smsLogGroup
	{
		smsLogRouter.DELETE("deleteSmsLog", smsLogApi.DeleteSmsLog)             // 删除短信日志
		smsLogRouter.DELETE("deleteSmsLogByIds", smsLogApi.DeleteSmsLogByIds)   // 批量删除短信日志
	}
	{
		smsLogRouterWithoutRecord.GET("getSmsLogList", smsLogApi.GetSmsLogList) // 获取短信日志列表
	}
}

File: backend/web/src/api/member/smsLog.js
import service from '@/utils/request';

// 分页获取短信日志列表
export const getSmsLogList = params => {
  return service({
    url: '/member/smsLog/getSmsLogList',
    method: 'get',
    params,
  });
};

// 删除短信日志
export const deleteSmsLog = params => {
  return service({
    url: '/member/smsLog/deleteSmsLog',
    method: 'delete',
    params,
  });
};

// 批量删除短信日志
export const deleteSmsLogByIds = params => {
  return service({
    url: '/member/smsLog/deleteSmsLogByIds',
    method: 'delete',
    params,
  });
};
